types/sinon/ts3.1/index.d.ts

```diff
@@ -1441,6 +1441,10 @@ declare namespace Sinon {
          * Requires the value to be a Symbol.
          */
         symbol: SinonMatcher;
+        /**
+         * Requires the value to be in the specified array.
+         */
+        in(allowed: any[]): SinonMatcher;
```

Comment:
may be worth using `unknown[]`, though it wouldn't make a difference other than one less `any` :p

Reply:
For arguments you pass it's pretty much irrelevant as `any` and `unknown` would be equivalent in this case, if it was the return value I'd agree.

Reply:
Of course, as I had already mentioned it makes no difference but means one day we could have a no-any lint rule on which is nice. Just personal preference